extensions/xcode/package.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "xcode",
   "title": "Xcode",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
As a new command has been added it would be good to bump the minor version number and set the version to `2.2.0`